### PR TITLE
Fix and improve the setup

### PR DESCRIPTION
**Pull request by PetarKirov**
See the individual commits for more details.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,21 +12,30 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
...
 
 # Download and run the install script
 RUN curl -fsS -o /tmp/install.sh https://dlang.org/install.sh
 RUN bash /tmp/install.sh -p /dlang install ${DLANG_VERSION}
 
+COPY ./har /tmp/har/src
+SHELL ["/bin/bash", "-c"]
+RUN source /dlang/$(ls -tr /dlang | tail -n1)/activate; \
+  echo $PATH && \
+  $DMD -of=/tmp/har/src/har -g -debug /tmp/har/src/harmain.d /tmp/har/src/archive/har.d
```
Comment (PetarKirov) [inline Dockerfile:32]: @wilzbach @Geod24  It's ugly but I didn't find a better way. Do you guys have any suggestions?

**Comment by PetarKirov** [comment]
Finally green :confetti_ball:

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM ubuntu:19.10
```
Comment (Geod24) [inline Dockerfile:1]: What's the reason for not using an LTS ?

Comment (PetarKirov) [reply Dockerfile:1]: To have access to newer packages.

**Pull request merged**